src/components/AgendaItemCard.tsx

```diff
@@ -282,3 +294,271 @@ export function SearchResultAgendaItemCard({
+                  <dt className="ml-auto">Motion</dt>
+                  <dd className="text-gray-500">{motion.motionType}</dd>
+                </dl>
+
+                <dl className="flex flex-row gap-4 justify-between items-center text-xs">
+                  <div className="flex items-center">
+                    <dt className="inline-flex items-center justify-center w-8 h-8 rounded-full bg-[#1870F8] text-white text-md">
+                      <VoteIcon value={motion.value} />
+                    </dt>
+                    <dd className="pl-2">
+                      <div>Vote</div>
+                      <div className="text-gray-500">{motion.value}</div>
+                    </dd>
+                  </div>
+
+                  <div className="flex flex-col items-center text-center">
+                    <dt>Decision body</dt>
+                    <dd className="text-gray-500">{motion.committeeName}</dd>
+                  </div>
+
+                  <div className="flex flex-col items-center text-center">
+                    <dt>Total</dt>
+                    <dd className="text-gray-500">{motion.tally}</dd>
+                  </div>
+
+                  <div className="flex flex-col items-center text-center">
+                    <dt>Status</dt>
+                    <dd className="text-gray-500">{motion.resultKind}</dd>
+                  </div>
+                </dl>
+              </div>
+            ))}
+          </div>
+        </div>
+      </AgendaItemCard>
+    </Link>
+  );
+}
+
+// Add the VoteIcon component at the top of the file
+const VoteIcon = ({ value }: { value: string }) => {
+  switch (value?.toLowerCase()) {
+    case 'yes':
+      return <ThumbsUpIcon className="w-4 h-4" />;
+    case 'no':
+      return <ThumbsDownIcon className="w-4 h-4" />;
+    case 'absent':
+      return <CircleMinusIcon className="w-4 h-4" />;
+    default:
+      return <CircleDotDashedIcon className="w-4 h-4" />;
+  }
+};
+
+// Add Councillor type before the SearchPageCouncillorCard
+export type Councillor = {
+  id: string;
+  name: string;
+  ward: string;
+  wardNumber: number;
+  email: string;
+  phone: string;
+  photoUrl: string;
+  keyIssues: string[];
+  profile: string;
+  stats: {
+    voted: number;
+    moved: number;
+    seconded: number;
+  };
+};
+
+export function SearchPageCouncillorCard({
+  councillor,
+  _decisionBody,
+}: {
+  councillor: Councillor;
+  _decisionBody: DecisionBody;
+}) {
+  return (
+    <Card className="transition-shadow hover:shadow-md">
+      <CardHeader>
+        <div>
+          <Chip variant="secondary" className="bg-blue-100 text-black">
+            Current Councillor
+          </Chip>
+        </div>
+      </CardHeader>
+      <CardContent>
+        <div className="space-y-4">
+          {/* Profile section */}
+          <div className="flex items-start gap-4">
+            <div className="relative w-16 h-16 rounded-full overflow-hidden">
+              <Image
```

Comment:
`src/components/Avatar.tsx` can be used for councillor avatars (centers the image on their head and handles the fallback automatically since a bunch have busted image links in the open data dataset)